### ISSUE TITLE
User management - forgot password

### ISSUE THREAD

**Issue by songwoo** [songwoo/tracker#24]
Not working.

**Comment by Pierroutt** [comment]
Should be working now, just tried it with my own username and it sent an email. It might be marked as spam, but I'm not sure.
What exactly was your specific issue by the way? Was it the email not being sent or the password not being reset?

**Comment by songwoo** [comment]
It's not working for me. I checked the spam folder.

When I create a new user, I am neither able to choose a password or to have
the system send an email.

When I choose forgot password and enter my email address, nothing happens.
Also, the screen should not look like this. It should be a standalone
screen with no other elements.

[image: Inline image 1]

_-------------------_
_Woo Song_
+1.917.410.1123
songwoo@gmail.com
http://www.linkedin.com/in/songwoo

**Comment by Pierroutt** [comment]
What exactly do you mean by "when I create a new user, I am neither able to choose a password or to have the system send an email."? What exactly do you mean by choose a password?
Also...I have to ask again, because I just tested this, but are you completely sure it's not working?
I created another account using another gmail account I have and I can confirm that I got an email from xogene asking to click a link to reset a password.
It's titled System, and the sender is no-reply@bcd.xogene.com (I'm trying to fix this to say tracker.xogene.com).

**Comment by songwoo** [comment]
I need to show you . Are you available for a quick uberconference?

_-------------------_
_Woo Song_
+1.917.410.1123
songwoo@gmail.com
http://www.linkedin.com/in/songwoo

**Comment by Pierroutt** [comment]
Apologies yesterday for not being available, but I am available now for an uberconference..

**Comment by songwoo** [comment]
Let's do this now.

_-------------------_
_Woo Song_
+1.917.410.1123
songwoo@gmail.com
http://www.linkedin.com/in/songwoo

**Comment by Pierroutt** [comment]
oh god I am so sorry, i forgot to check the github pages, I assumed you were gonna email me so I just kept refreshing my email.

**Comment by Pierroutt** [comment]
I am absolutely ready now if you want to discuss it. Although, to be completely honest, I don't really know how if I can fix this issue, partially because I did not write this portion of the code, but mostly because I tested it again and it worked for my third gmail email. I've been going over the code all day long to try and understand it but I cannot find a single real mistake inside of it. If you really cannot find the email inside of your inbox, I think it has to do with how whichever email server is handling your email (although if you're using gmail, I really don't know what's going on, because the three times I tested this, I used gmail twice, and yahoo once).
